dandi/metadata/util.py

```diff
@@ -394,6 +394,12 @@ def extract_cellLine(metadata: dict) -> str | None:
         NCBITAXON_URI_TEMPLATE.format("9545"),
         "Macaca nemestrina",
     ),
+    (
+        ["Mongolian gerbil", "Mongolian jird"],
```

Comment:
These need to be lowercase.

Reply:
```suggestion
        ["mongolian gerbil", "mongolian jird"],
```

Reply:
ideally IMHO we should make code robust to specification here and just `.lower()` then when matching. Or do you see an issue with that @jwodder ?